transifex/pull.py

```diff
@@ -223,7 +229,7 @@ def pull(repo):
                 logger.info('No changes detected for [%s]. Cleaning up.', repo.name)
 
         if pr:
-            if not repo.is_compiled:
+            if (not skip_compile_messages) and (not repo.is_compiled):
```

Comment:
Minor: for clarity, consider using the existing `self.is_compiled` a little differently. You could just default it to False, for example, only setting it True once compilation completes, if it's run at all.